config/config.example.json

```diff
@@ -48,7 +48,9 @@
     "jobs": {
         "updateServerCount": {
             "schedule": "0 */10 * * * *",
-            "log": false
+            "log": false,
+            "runOnce": false,
```

Comment:
`runOnce` and `initialDelaySeconds` aren't needed now since they're the defaults

Reply:
see above